pkg/constants/constants.go

```diff
@@ -46,6 +46,10 @@ const (
 
 	QUAY_OAUTH_TOKEN_RELEASE_DESTINATION string = "QUAY_OAUTH_TOKEN_RELEASE_DESTINATION"
 
+	PYXIS_STAGE_KEY string = "PYXIS_STAGE_KEY"
+
+	PYXIS_STAGE_CERT string = "PYXIS_STAGE_CERT"
+
```

Comment:
```suggestion
	PYXIS_STAGE_KEY_ENV string = "PYXIS_STAGE_KEY"

	PYXIS_STAGE_CERT_ENV string = "PYXIS_STAGE_CERT"

```
+ please add a comment describing what these env vars are used for